taskfile.yaml

```diff
@@ -221,6 +228,15 @@ tasks:
       vars:
         COMPONENT: "{{.TASK}}"
 
+  generate-config-schemas:
+    vars:
+      OUTPUT_DIR: "{{.G_CONFIG_SCHEMA_BUILD_DIR}}"
+    cmds:
+      - |-
+        rm -rf "{{.OUTPUT_DIR}}"
+        mkdir "{{.OUTPUT_DIR}}"
```

Comment:
we should use `mkdir -p` to create any missing parent paths because the `build` directory may not be available when the task is run

otherwise, `mkdir` may complain that the path does not exist

@coderabbitai create an issue to track